src/cppcache/src/MapSegment.hpp

```diff
@@ -164,9 +166,9 @@ class CPPCACHE_EXPORT MapSegment {
     m_entryFactory->newMapEntry(key, newEntry);
     newEntry->setValueI(newValue);
     if (m_concurrencyChecksEnabled) {
-      if (versionTag != NULLPTR && versionTag.ptr() != NULL) {
+      if (versionTag != NULLPTR && versionTag.ptr() != nullptr) {
```

Comment:
The original `NULL` is C for `0` and was allowed in C++ as an implicit cast to `(void *)0`.